src/js_user.hpp

```diff
@@ -242,18 +242,23 @@ void UserClass<T>::link_credentials(ContextType ctx, ObjectType this_object, Arg
 
 template<typename T>
 void UserClass<T>::call_function(ContextType ctx, ObjectType this_object, Arguments& args, ReturnValue &) {
-    args.validate_count(3);
+    args.validate_count(4);
```

Comment:
are all arguments required. Isn't something optional

Reply:
Optional in the user-facing JS API. Not optional in the internal C++ API. Well semi-optional since it accepts an explicit `undefined`.

Reply:
👍